components/pluralrules/src/operands.rs

```diff
@@ -70,6 +71,27 @@ pub struct PluralOperands {
     pub t: u64,
 }
 
+/// PartialEq is implemented explicitly instead of derived, to account for float comparison in the
+/// internal representation.
```

Comment:
IIUC, *n* isn't actually used for plural rules resolution.  Right, @zbraniecki?  Maybe we should remove it from PluralOperands.  In particular, converting from a FixedDecimal to a double will be slow and imprecise.

Reply:
IMHO it would have been easier if the fields weren't exposed on the `PluralOperands`.  We'd be able to construct `n` lazily.  But it's exactly the spec.

Reply:
We can still change `PluralOperands`.  Nothing has been released yet.

I kind-of like the idea of making a helper method on PluralOperands that calculates `n` on demand.

Reply:
There's a potentially simpler way. *If* we can assume an invariant that the value of `n` always corresponds the other fields (which technically we can not since all fields are public), we could simply remove the comparison between `self.n` and `other.n` from this `eq` implementation, and we'd still be correct.

Reply:
Interestingly enough, with my implementation of `eq`, the ostensibly more complicated implementation performs *better* than the ostensibly less complicated one.

Unless you are specifically worried about something in this code, I'd keep it as is for now.  It seems internal enough to be painless to improve later.